bodhi/server/validators.py

```diff
@@ -496,16 +523,16 @@ def validate_acls(request, **kwargs):
 
 
 @postschema_validator
-def validate_uniqueness(request, **kwargs):
+def validate_build_uniqueness(request, **kwargs):
```

Comment:
I'm fine with renaming this, but it seems unrelated to the rest of the PR - can we do this in a separate pull request?

Reply:
Sure, I'll put this and the two type-change ones in their own PR (one PR for the three commits).